app/js/components/common/Header.jsx

```diff
@@ -55,15 +55,15 @@ export default class Header extends Component {
     this.fetchLocation('/session').then((response) => {
       this.setState((prevState, props) => {
         return {
-          currentUser: response.user.display
+          currentUser: response.user && response.user.display
```

Comment:
I love the fact that you are checking if the `user` value is not null before setting the state. But I think you should consider if `response.user` is `null` as well except if you don't want the state to change if it's null. `response.user ? response.user.display : ''` should do the job for you.

Nice work though 👍

Reply:
Thanks @andela-dbamidele for the nitpick. Would put this into effect.